### PR TITLE
Fix broken link in examples

### PR DESCRIPTION
**Pull request by jede**
Just found a broken link :)

### CODE REVIEW

File: examples/index.md [hunk 1]
```diff
@@ -275,7 +275,7 @@ Content-Type: application/vnd.api+json
 ```
 
 > Note: in the responses above with a 422 status code, `400 Bad Request` would
-also be acceptable. ([http://stackoverflow.com/a/20215807/1261879](More details.))
+also be acceptable. [More details.](http://stackoverflow.com/a/20215807/1261879)
```
Comment (bf4) [inline examples/index.md]: weird, that should have rendered correctly. I guess `[(More details.)](http://stackoverflow.com/a/20215807/1261879)` was what I wanted.

[(More details.)](http://stackoverflow.com/a/20215807/1261879)

Comment (bf4) [inline examples/index.md]: Oh, nm I just got it backwards
- `([More details.](http://stackoverflow.com/a/20215807/1261879))`
- ([More details.](http://stackoverflow.com/a/20215807/1261879))

Comment (steveklabnik) [inline examples/index.md]: Yeah. @jede can you re-add the external parenthesis?

Comment (jede) [inline examples/index.md]: Oh, sorry. Better?

Comment (jede) [inline examples/index.md]: Oh, sorry. Better now?

**Comment by steveklabnik** [comment]
Thanks so much!

**Pull request merged**